pkg/apis/core/v1alpha1/zz_generated.kubebuilder.go

```diff
@@ -283,7 +283,6 @@ var (
 			Scope: "Namespaced",
 			Validation: &v1beta1.CustomResourceValidation{
 				OpenAPIV3Schema: &v1beta1.JSONSchemaProps{
-					Type: "object",
```

Comment:
Apologies for my ignorance, but why is this change needed as part of this PR?

Reply:
The generated code by kubebuilder for types containing status sub-resource has issue during validation. So the work around is to skip that validation for now by manually fixing. this issue should go away once the pr https://github.com/kubernetes/kubernetes/pull/65357 is cherry-picked to 1.11 branch or we move to 1.12 branch. 
here https://github.com/kubernetes/kubernetes/pull/65357#issuecomment-401919242 is more details on that conversation